CRM/Core/BAO/MessageTemplate.php

```diff
@@ -389,6 +389,11 @@ public static function sendTemplate($params) {
 
     CRM_Utils_Hook::alterMailParams($params, 'messageTemplate');
 
+    // Core#644 - handle contact ID passed as "From".
```

Comment:
I know this commit has already been merged, but I feel like this block where the FROM address is "fixed" should be before the alterMailParams hook is called. otherwise we're sending bad data to that hook and then fixing it.

Reply:
@lcdservices That makes sense.  If you submit a PR I'll review it?

Reply:
@MegaphoneJon here you go: https://github.com/civicrm/civicrm-core/pull/13776